lib/api.ts

```diff
@@ -191,10 +197,27 @@ export function getCalendarData(activity: Activity[]) {
         };
       }
       acc[date].count += 1;
+
+      const uniquePrReviews = new Set();
```

Comment:
Could you explain why there are Two sets? All Unique and Unique? Could you explain the algorithm for how you are solving the problem?

Reply:
The AllUnq set is used to count the total unique pull requests (PRs) reviewed.

and the 2nd set is For PRs reviewed in the last 7 days, we calculate and count the reviews for each date (day) and sum them up over this period.

In this process, if a PR has multiple reviews on the same day or on subsequent days, it is only counted once for its first review. Any subsequent reviews of the same PR are not counted again.

Reply:
@konavivekramakrishna multiple distinct reviews on the same PR must be counted. But we shouldn't count the individual review comment on the PR. We should only count one review event.

Reply:
@rithviknishad 
For example, in the following pr: https://github.com/coronasafe/care_fe/pull/6479

what should be the count of pr_reviewed

The count of 'pr_reviewed' for this pull request in the current staging leaderboard is 13

Reply:
> @konavivekramakrishna multiple distinct reviews on the same PR must be counted. But we shouldn't count the individual review comment on the PR. We should only count one review event.

@rithviknishad are you referring to multiple individuals reviewing the same pull request, or do you mean counting multiple distinct reviews by a single person?"

Reply:
No. For example, the recent review I've made on your PR in care (https://github.com/coronasafe/care_fe/pull/7152#pullrequestreview-1858666162), I've only requested changes ONCE (and you might have also got a mail notification once for the same). However, leaderboard shows 2 review events even though I've made only 1 review event.

<img width="1175" alt="image" src="https://github.com/coronasafe/leaderboard/assets/25143503/678c39cb-6619-46f9-90e5-e52775728f6b">


<img width="1428" alt="image" src="https://github.com/coronasafe/leaderboard/assets/25143503/751821f5-5c97-4ccf-87e2-8a6356576851">

Reply:
The review event seems to be having an ID. Could you update the types? Or if we are not scrapping the ID, can we adjust the scraper to report only one review event for an ID?